Source/Core/Core/ConfigManager.cpp

```diff
@@ -381,6 +382,15 @@ void SConfig::LoadGeneralSettings(IniFile& ini)
 
   general->Get("NANDRootPath", &m_NANDPath);
   File::SetUserPath(D_WIIROOT_IDX, m_NANDPath);
+  general->Get("DumpPath", &m_DumpPath);
+  if (!m_DumpPath.empty())
+  {
+    File::SetUserPath(D_DUMP_IDX, m_DumpPath + '/');
+    File::CreateDir(File::GetUserPath(D_DUMPAUDIO_IDX));
```

Comment:
`CreateDir` only creates the last path segment; can we be sure that `D_DUMP_IDX` has been created by now?